pkg/drivers/drivers.go

```diff
@@ -136,3 +150,88 @@ func fixPermissions(path string) error {
 	}
 	return nil
 }
+
+// Download downloads driver if it is not present, or if the version is old
+func Download(driver, destination string, minikubeVersion semver.Version) error {
```

Comment:
- it is called by the start cmd (https://github.com/kubernetes/minikube/pull/5085/files#diff-ed78cd81495af3964aba5fbc898c33a2R941), we don't add callers in comments for no other code, so I will prefer to avoid that here too.
- great idea about the name